test/gui/tst_sharing/test.feature

```diff
@@ -140,16 +140,33 @@ Feature: Sharing
         Given user "Alice" has created on the server folder "simple-folder"
         And user "Alice" on the server has created file "lorem.txt"
         And user "Brian" has been created on the server with default attributes and without skeleton files
-        And user "Alice" has shared folder "simple-folder" on the server with user "Brian"
-        And user "Alice" has shared folder "lorem.txt" on the server with user "Brian"
+    	And user "Alice" on the server has shared folder "simple-folder" with user "Brian" with "all" permissions
+    	And user "Alice" on the server has shared file "lorem.txt" with user "Brian" with "all" permissions
         And user "Alice" has set up a client with default settings
         When the user removes permissions "<permissions>" for user "Brian Murphy" of resource "%client_sync_path%/simple-folder" using the client-UI
+        And the user closes the sharing dialog
```

Comment:
indents seem to be a bit messed up, maybe your IDE gets confused with tabs and spaces?

Reply:
fixed